docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+# Use the default network olog_default.
+version: "3.9"
+services:
+  olog:
+    build: .
+    ports:
+      - "8181:8181"
+    depends_on:
+      - mongo
+      - elastic
+    # Ensure that the elastic server is up before starting olog.
+    command: ["wait-for-it", "-t", "30", "elastic:9200", "--", "java", "-jar", "target/olog-es-2.0.jar", "--spring.config.name=application-docker"]
+  mongo:
+    image: mongo
+    ports:
+      - 27017:27017
+    volumes:
+      - olog-mongodata:/etc/mongo
+  elastic:
+    image: elasticsearch:6.8.13
+    environment:
+      - cluster.name=elasticsearch
+      # I see errors with this set to true.
```

Comment:
I am not experiencing this. In our compose we use `elasticsearch:6.8.4` like the README.md says, and `bootstrap.memory_lock: true` works for me.